Tests/test_KDTree.py

```diff
@@ -195,7 +195,8 @@ def test_KDTree(self):
             self.assertTrue(test(nr_points, dim, bucket_size, radius))
 
     def test_all_search(self):
-        self.assertTrue(test_all_search(nr_points, dim, bucket_size, query_radius))
+        for i in range(0, 5):
+            self.assertTrue(test_all_search((nr_points / 10), dim, bucket_size, query_radius))
```

Comment:
Use explicit integer division here (``//`` not ``/``) in order to work as expected on Python 2 and 3